src/core/jsonschema/jsonschema.cc

```diff
@@ -438,9 +457,12 @@ auto sourcemeta::core::vocabularies(const SchemaResolver &resolver,
   if (!result.contains(core)) {
```

Comment:
Can we take advantage of your bitsets here? See that we get the core vocabulary for a dialect using `core_vocabulary()`. That one could return a known vocabulary enum member, and thus check containment using the bit maps?